en/reference/introduction.rst

```diff
@@ -81,14 +74,17 @@ Then run the following commands on your command line
     $ curl -s http://getcomposer.org/installer | php --
     $ php composer.phar install
 
-This will download the dependencies into the vendor/ folder and generate vendor/autoload.php
+This will download the dependencies into the vendor/ folder and generate ``vendor/autoload.php``.
 
 .. _intro-bootstrap:
 
 Now we bootstrap Doctrine PHPCR-ODM. Create a file called ``bootstrap.php`` in
-your project root directory::
+your project root directory:
+
+.. code-block:: php
```

Comment:
is php not the default code language? if not, we could make it that, then it would work the same as with the symfony doc.